overlays/tls-overlay.yaml

```diff
@@ -0,0 +1,11 @@
+---
+applications:
+  ca:
+    charm: self-signed-certificates
+    channel: edge
+    scale: 1
+    options:
+      ca-common-name: ca.demo.local
```

Comment:
I wonder if we should have it for the official bundle or leave it to the end users to configure.

Reply:
I see your point.
I thought of keeping it coherent with what we have in https://github.com/canonical/cos-lite-bundle/blob/main/overlays/tls-overlay.yaml#L7